recipes/psyinf-gmtl/all/conanfile.py

```diff
@@ -39,8 +39,11 @@ def package(self):
     def package_info(self):
         self.cpp_info.bindirs = []
         self.cpp_info.libdirs = []
+        self.cpp_info.names["cmake_find_package"] = "gmtl"
+        self.cpp_info.names["cmake_find_package_multi"] = "gmtl"
+        
         self.cpp_info.set_property("cmake_file_name", "gmtl")
-        self.cpp_info.set_property("cmake_target_name", "gmtl")
+        self.cpp_info.set_property("cmake_target_name", "gmtl::gmtl")
```

Comment:
It doesn't follow upstream convention where target is exported without namespace: https://github.com/psyinf/gmtl/blob/67c9f5798ba165403b52e39903a229fc7f749f14/CMakeLists.txt#L54-L59

Reply:
Sorry, I'm not an expert in CMake unfortunately. What should I do to fix this? Are changes to the CMakeLists.txt required? 
So so I need to set NAMESPACE ${PROJECT_NAME}:: here? 
All other packages I've encountered so far seem to use the prefix

Reply:
conan-center recipes follow upstream convention. Didn't you write upstream CMakeLists?

You can either fix upstream CMakeLists by exporting target with a namespace gmtl (see NAMESPACE argument of https://cmake.org/cmake/help/latest/command/install.html#installing-exports). In this case yes we can define the target as gmtl::gmtl

Or upstream CMakeLists is kept as is, and therefore you have to provide a `gmtl` target in cmake_find_package[_multi] generator (see `ade` recipe for example), and no need to change `cmake_target_name` property anymore.

Reply:
The original CMakeLists was authored by an colleague of mine. I'll explore the options, but I think changing the CMakeLists is the way to go (as it needs changes anyways to contain the corrected version)

Reply:
I've changed the CMakeLists (hopefully correctly).  Can you please review, if this is now conforming?

Reply:
yes.

By the way you should remove `export(PACKAGE ${PROJECT_NAME})`, because:
- target is not exported in build tree anyway
- usually user don't like such side effect (see https://cmake.org/cmake/help/latest/policy/CMP0090.html)

Reply:
I'll address the rest of the CMake issues with the next revision